igneous/task_creation.py

```diff
@@ -278,12 +278,8 @@ def create_deletion_tasks(task_queue, layer_path, mip=0, num_mips=5):
   })
   vol.commit_provenance()
 
-
-def create_meshing_tasks(
-    task_queue, layer_path, mip, 
-    shape=Vec(512, 512, 64), max_simplification_error=40,
-    mesh_dir=None, cdn_cache=False 
-  ):
+def create_meshing_tasks(task_queue, layer_path, mip, shape=Vec(512, 512, 256),
```

Comment:
- We seem to have lost `cdn_cache` and `mesh_dir`
- Less important, but it'd be nice to maintain the whitespace organization.

Reply:
good catch, fixed this.